docs/signup/openapi.yaml

```diff
@@ -184,40 +187,46 @@ definitions:
         description: Vipps sale unit's merchant serial number
       organizationNumber:
         type: string
-        pattern: '^\d{9}$'
-        minLength: 9
-        maxLength: 9
+        pattern: '^\d{6}$'
+        minLength: 6
+        maxLength: 6
+        example: "123456"
```

Comment:
Norwegian organisation number is 9 digit. For now, we would require PSPs to be able to provide a valid Norwegian organisation number.